app/components/poll/index.js

```diff
@@ -0,0 +1,194 @@
+    backgroundColor: theme.palette.info.light,
+  },
+}))(LinearProgress);
+const SuccessProgressBar = withStyles((theme) => ({
+  bar: {
+    backgroundColor: theme.palette.success.light,
+  },
+}))(LinearProgress);
+
+const useStyles = makeStyles(() => ({
+  spacing: 1,
+  paper: {
+    padding: '30px',
+    borderRadius: '5px',
+    marginRight: '30px',
+  },
+}));
+
+export const Poll = ({
+  id,
+  name,
+  description,
+  firstOption,
+  secondOption,
+  thirdOption,
+  fourthOption,
+  firstOptionVotes,
+  secondOptionVotes,
+  thirdOptionVotes,
+  fourthOptionVotes,
+  handleChange,
+  radioValue,
+}) => {
+  const [hidden, setHidden] = useState(false);
+  const classes = useStyles();
+  const history = useHistory();
+
+  const navigateTo = (url) => {
+    history.push(url);
+  };
+  const handleDeleteEvent = () => {
+    Modal.fire();
+  };
+  return (
+    <>
+      <Box>
+        <Box mr={10} mb={2} my={3} display="flex" justifyContent="flex-end">
+          <IconButton>
+            <EditIcon
+              color="secondary"
+              onClick={() => navigateTo(`/polls/edit/${id}`)}
+            />
+          </IconButton>
+          <IconButton>
+            <DeleteIcon color="error" onClick={handleDeleteEvent} />
+          </IconButton>
+        </Box>
+        <Paper elevation={1} className={classes.paper}>
+          <Box my={7}>
+            <H5>{name}</H5>
+          </Box>
+          <Box>
+            <BodyTextLarge bold> {description}</BodyTextLarge>
+          </Box>
+
+          <RadioGroup
+            aria-label="poll"
+            name="poll"
+            value={radioValue}
+            onChange={handleChange}
+          >
+            <FormControlLabel
+              value={firstOption}
+              control={<Radio />}
+              label={firstOption}
+            />
+            <FormControlLabel
+              value={secondOption}
+              control={<Radio />}
+              label={secondOption}
+            />
+            <FormControlLabel
+              value={thirdOption}
+              control={<Radio />}
+              label={thirdOption}
+            />
+            <FormControlLabel
+              value={fourthOption}
+              control={<Radio />}
+              label={fourthOption}
+            />
+          </RadioGroup>
+
+          <Box display="flex" flexDirection={['column', 'row', 'row']}>
+            <Box mr={4} my={3}>
+              <Button variant="contained" color="secondary">
+                Vote
+              </Button>
+            </Box>
+            <Box my={3}>
+              {hidden ? (
+                <Button variant="contained" onClick={() => setHidden(false)}>
+                  Show Results
+                </Button>
+              ) : (
+                <Button variant="contained" onClick={() => setHidden(true)}>
+                  Hide Results
+                </Button>
+              )}
+            </Box>
+          </Box>
+
+          {hidden ? null : (
```

Comment:
change this condition to !hidden && ( <results...> )

Reply:
resolved